server/lib/transforms/html/table.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const match = require('@quarterto/cheerio-match-multiple');
+
+function generateTableHref(el) {
+	const tableId = el.attr('id');
+	const contentId = tableId.match(/table-(.*)_\d/)[1];
+
+	return `https://ft.com/content/${contentId}#${tableId}`;
```

Comment:
We expect to always have those information but maybe we could be more defensive and prevent to have a broken link. While the tableId is probably not so critical, the contentId it is critical for the href.

```suggestion
	return contentId ? `https://ft.com/content/${contentId}#${tableId}` : false;
```
As a fallback, instead of a broken link we show the table (or potentially nothing)